web_src/js/index.js

```diff
@@ -1184,6 +1184,12 @@ async function initRepository() {
     // Change status
     const $statusButton = $('#status-button');
     $('#comment-form textarea').on('keyup', function () {
+      const $simplemde = $(this).data('simplemde');
+      if ($simplemde && $simplemde.value().length !== 0) {
+        $statusButton.text($statusButton.data('status-and-comment'));
+        return;
+      }
+
```

Comment:
Try this more compact solution for this handler:

```js
const $simplemde = $(this).data('simplemde');
const value = $simplemde ? $simplemde.value() : $(this).val();
$statusButton.text($statusButton.data(value.length === 0 ? 'status' : 'status-and-comment'));
```

Reply:
tested, didnt work ...?

Reply:
Hmm maybe this:

```js
const $simplemde = $(this).data('simplemde');
const value = ($simplemde && $simplemde.value()?.length) ? $simplemde.value() : $(this).val();
$statusButton.text($statusButton.data(value.length === 0 ? 'status' : 'status-and-comment'));
```

Reply:
@silverwind you are right, the first solution works well, if you replace the code below too ...

Reply:
sorry ... didn't read the whole func and wat it does until now :swan:

Reply:
Of course I meant the whole function, that's why I wrote "this handler" 😉